Controller/NotifyHandler.php

```diff
@@ -87,16 +87,29 @@ public function handleRequest()
 
         $oTransaction = oxNew(Transaction::class);
 
-        if ($oTransaction->loadWithTransactionId($oNotificationResponse->getTransactionId())) {
+        if ($oTransaction->loadWithTransactionId($oNotificationResp->getTransactionId())) {
             // if a transaction with this ID already exists, we do not need to handle it again
             return;
         }
 
+        $this->_handleNotificationResponse($oNotificationResp, $oService);
+    }
+
+    /**
+     * Handles the success and error response coming from the paymentSDK.
+     *
+     * @param Response       $oNotificationResp
+     * @param BackendService $oService
```

Comment:
I would call it `$oBackendService` instead to make it more descriptive.

Reply:
👍